### PR TITLE
skipPush -> push

### LINKED ISSUE #1113: Rename skipPush -> push
`skipPush` is a negated field, can we change it to `push`?  

`skipPush: false` -> `push: true`
`skipPush: true` -> `push: false`

### PR DESCRIPTION
**Pull request by balopat**
Fixes #1113.

### CODE REVIEW

**Comment by balopat** [comment]
Thanks for catching that @dgageot! Done.

File: skaffold.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: skaffold/v1alpha3
 kind: Config
 build:
   artifacts:
-  - image: gcr.io/k8s-skaffold/skaffold
+  - imageName: gcr.io/k8s-skaffold/skaffold
     docker:
       dockerfile: deploy/skaffold/Dockerfile
```
Comment (dgageot) [inline skaffold.yaml]: I think `dockerfile` should be `dockerfilePath`

Comment (balopat) [reply skaffold.yaml]: agreed - I changed it

File: integration/examples/annotated-skaffold.yaml [hunk 2]
```diff
@@ -77,7 +77,8 @@ build:
   # `useBuildkit` can also be set to activate the experimental BuildKit feature.
   #
   # local:
-  #   skipPush: true
+  #   false by default for local clusters, true for remote clusters
+  #   push: false
```
Comment (thomas-riccardi) [inline integration/examples/annotated-skaffold.yaml]: The doc in the previous paragraph still talks about skip push, with inverted boolean value: this is confusing.

**Pull request merged**